rtloader/include/rtloader_types.h

```diff
@@ -125,6 +125,8 @@ typedef void (*cb_set_external_tags_t)(char *, char *, char **);
 typedef void (*cb_write_persistent_cache_t)(char *, char *);
 // (value)
 typedef char *(*cb_read_persistent_cache_t)(char *);
+// (value)
```

Comment:
we try to explain what is the parameter here. It's the "value" when doing a write to the cache. The comment for obfuscate_sql could be `// (sql_query)`

Reply:
Updated